### PR TITLE
Expose barcode's location

### PR DESCRIPTION
**Pull request by lucatorella**
- Barcode’s `location` and `convertedLocation`
- TrackedBarcode’s `predictedLocation` and `convertedPredictedLocation`

### CODE REVIEW

File: src/ios/SBSTypeConversion.m [hunk 1]
```diff
@@ -38,6 +57,13 @@ - (long)uniqueId {
                                  [NSNumber numberWithBool:[code isGs1DataCarrier]], @"gs1DataCarrier",
                                  [NSNumber numberWithBool:[code isRecognized]], @"recognized", nil];
     [dict setObject:@(code.compositeFlag) forKey:@"compositeFlag"];
+    [dict setObject:SBSJSObjectFromQuadrilateral(code.location, picker, NO) forKey:@"location"];
+    [dict setObject:SBSJSObjectFromQuadrilateral(code.location, picker, YES) forKey:@"convertedLocation"];
+    if ([code isKindOfClass:[SBSTrackedCode class]]) {
+        SBSQuadrilateral predictedLocation = ((SBSTrackedCode *)code).predictedLocation;
+        [dict setObject:SBSJSObjectFromQuadrilateral(predictedLocation, picker, NO) forKey:@"predictedLocation"];
+        [dict setObject:SBSJSObjectFromQuadrilateral(predictedLocation, picker, YES) forKey:@"convertedPredictedLocation"];
```
Comment (petradonka) [inline src/ios/SBSTypeConversion.m]: neither of the converted locations are actually converted, right? 🤔

Comment (lucatorella) [reply src/ios/SBSTypeConversion.m]: why not?

Comment (lucatorella) [reply src/ios/SBSTypeConversion.m]: the last argument is triggering the conversion

Comment (petradonka) [reply src/ios/SBSTypeConversion.m]: sorry, a combination of github diffs and probably not paying enough attention 🙄

**Pull request merged**